### ISSUE TITLE
Fastparquet coverage

### ISSUE THREAD

**Issue by mrocklin** [dask/dask#2896]
Looking briefly at our test coverage it looks like the fastparquet components of `dask/dataframe/io/parquet.py` are uncovered.  Inspecting `continuous_integration/travis/install.sh` it looks like we avoid fastparquet for newer versions of NumPy.  Is this still necessary?

```
if [[ $PYTHONOPTIMIZE != '2' ]] && [[ $NUMPY > '1.11.0' ]] && [[ $NUMPY < '1.13.0' ]]; then
    conda install -q -c conda-forge numba cython
    pip install -q --no-deps git+https://github.com/dask/fastparquet
fi
```

**Comment by jcrist** [comment]
Just to be clear, they're still tested, but just not in the build that does coverage (which is py27 only). In any case, I just tested and all tests pass with numpy 1.13.3.

**Comment by jcrist** [comment]
Per this build (https://travis-ci.org/dask/dask/jobs/303585408) it's installed, but the tests skip. Perhaps a missing dependency causes imports to fail?

**Comment by jcrist** [comment]
Should be fixed in #2897.
